### DISCUSSION TITLE
Add a GitHub action that compares the code to the latest upstream VASM source

### DISCUSSION THREAD

**Discussion by mbitsnbites** [mbitsnbites/vasm-mirror#16] [Ideas]
It should be possible to make sure that the code in this repo is identical to the latest upstream VASM source (except for a few GitHub-specific files, like README.md and LICENSE).

It would be nice to perform this check as part of the PR, via GitHub actions.

**Comment by StarWolf3000** [comment]
There's a daily snapshot of the source on the site, which is uploaded there from some private CVS repository, so the GH Action would have to do this:
1) Daily check for an update at certain times of the day (probably early in the morning at UTC, since the current latest is from 02:14 AM UTC)
2) Clone the mirror on master branch
2) Download the latest snapshot
3) Unpack the latest snapshot with overriding the mirror
4) Do a comparison

The problem I see here is if files/folders are no longer in the newer snapshot, the mirror would not know that they got deleted.
